.coderabbit.yaml

```
@@ -79,3 +79,12 @@ style:
     exclude:
       - "tests/**"
       - "docs/**"
+
+post_review_actions:
+  - name: "Send ntfy notification"
+    run: |
+      curl -H "Title: Code Review Complete" \
+           -H "Priority: default" \
+           -d "CodeRabbit has completed reviewing PR #${PR_NUMBER}" \
+           ntfy.sh/focusfeed_alerts
+           
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)